lms/static/sass/base/_variables.scss

```diff
@@ -134,6 +134,72 @@ $green-u1: desaturate($green,15%);
 $green-u2: desaturate($green,30%);
 $green-u3: desaturate($green,45%);
 
+// copied from cms/static/sass/_variables.scss
+$yellow: rgb(255, 252, 221);      // yellow color used by LMS
+//$yellow: rgb(237, 189, 60);     // yellow color used by Studio
+$yellow-l1: tint($yellow,20%);
+$yellow-l2: tint($yellow,40%);
+$yellow-l3: tint($yellow,60%);
+$yellow-l4: tint($yellow,80%);
+$yellow-l5: tint($yellow,90%);
+$yellow-d1: shade($yellow,20%);
+$yellow-d2: shade($yellow,40%);
+$yellow-d3: shade($yellow,60%);
+$yellow-d4: shade($yellow,80%);
+$yellow-s1: saturate($yellow,15%);
+$yellow-s2: saturate($yellow,30%);
+$yellow-s3: saturate($yellow,45%);
+$yellow-u1: desaturate($yellow,15%);
+$yellow-u2: desaturate($yellow,30%);
+$yellow-u3: desaturate($yellow,45%);
+
+// copied from cms/static/sass/_variables.scss
+$blue: rgb(0, 120, 176);    // blue color used by LMS
+//$blue: rgb(0, 159, 230);  // blue color used by Studio
```

Comment:
We should use the LMS blue here instead of studio. It is a bit odd that it varies across applications currently, but we can clean that up eventually. 

For the colors section, I think it is also ok to drop the // copied from... note.